content/applications/finance/accounting/receivables/customer_invoices/electronic_invoicing.rst

```diff
@@ -0,0 +1,82 @@
+some governments for fiscal control or to facilitate the administration.
+
+Any business that processes large volumes of documents and forms can benefit from an EDI solution to
+increase automation and improve accuracy.
+
+Odoo supports, among others, the following formats.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Format Name
+     - Applicability
+   * - Factur-X (CII)
+     - Default format on Odoo (enabled by default)
+   * - Peppol BIS Billing 3.0 (UBL)
+     - For companies which countries are part of the `EAS list
+       <https://docs.peppol.eu/poacc/billing/3.0/codelist/eas/>`_
+   * - E-FFF
+     - For Belgian companies
+   * - XRechnung (UBL)
+     - For German companies
+   * - Fattura PA (IT)
+     - For Italian companies
+   * - CFDI (4.0)
+     - For Mexican companies
+   * -  Peru UBL 2.1
+     - For Peruvian companies
+   * - SII IVA Llevanza de libros registro (ES)
+     - For Spanish companies
+
+.. seealso::
+   - :doc:`../../fiscal_localizations/overview/fiscal_localization_packages`
+   - :doc:`../../fiscal_localizations/overview/localizations_list`
+
+Configuration
+=============
+
+Go to :menuselection:`Accounting --> Configuration --> Journals --> Customer Invoices -->
+Advanced Settings --> Electronic Invoicing` and enable the formats you need for this journal.
+
+.. image:: electronic_invoicing/formats.png
+   :align: center
+   :alt: Select the EDI format you need
+
+Once an electronic invoicing format is enabled, XML documents are generated when clicking on
+:guilabel:`Confirm` in documents such as invoices, credit notes, etc. These documents are either
+visible in the attachment section, or embedded in the PDF.
+
+.. note::
+   - For E-FFF, the xml file only appears after having generated the PDF (:guilabel:`Print` or
+     :guilabel:`Send & Print` button), since the PDF needs to be embedded inside the xml.
+   - By default, the :guilabel:`Factur-X` option is enabled. It means that an XML file is
+     automatically included in the PDF document that is sent.
+   - The formats available depend on the country registered in your company's :guilabel:`General
+     Information`.
+   - Odoo supports the **Peppol BIS Billing 3.0** format that can be used via existing access
+     points.
+
+Adding formats available from July 2022 in a database created prior to July 2022
+--------------------------------------------------------------------------------
+
+If your database was created before July 2022 and you wish to use one of the formats available from
+July 2022, you have to install the module :guilabel:`Import/Export electronic invoices with UBL/CII`
+(module’s technical name: `account_edi_ubl_cii`) and your country-specific module.
+
+.. example::
+   If you want to use the Belgian format E-FFF in a database created prior to July 2022, you need to
+   install the Belgian module `l10n_be_edi` and the `account_edi_ubl_cii`.
```

Comment:
write the modules' name like you did above (normal name + technical name). You could even put them in a very short list like we did in other localization pages.